pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -626,7 +627,7 @@ func runHandler(ctx *domainContext, key string, c <-chan interface{}) {
 // Check if it is still running
 // XXX would xen state be useful?
 func verifyStatus(ctx *domainContext, status *types.DomainStatus) {
-	domainId, err := xlDomid(status.DomainName, status.DomainId)
+	domainID, err := xlDomid(status.DomainName, status.DomainId)
```

Comment:
Did yetus require the change to domainid capitalization or are you doing this just to make the reviewers life more painful ;-)

Reply:
I believe it was Yetus -- yes ;-)

It has to have at least one sacrifice in every commit -- this is the one @nordmark

Reply:
Yetus cribbed about it.
Along with "Id" is changed to "ID"
"Url" Changed to "URL"

Reply:
That's fine -- don't worry about it @spatchineelam -- lets keep yetus happy for now (even if it may anger @nordmark  ;-))